platform/src/main/resources/config/clients-schema.yaml

```diff
@@ -0,0 +1,87 @@
+# Examples of client configurations:
+# custodial_clients:
+#   - name: circle
+#     type: custodial
+#     signing_key: GBI2IWJGR4UQPBIKPP6WG76X5PHSD2QTEBGIP6AZ3ZXWV46ZUSGNEGN2
+#     callback_url: https://callback.circle.com/api/v1/anchor/callback
+# noncustodial_clients:
+#   - name: lobstr
+#     type: noncustodial
+#     domain: lobstr.co
+#     callback_url: https://callback.lobstr.co/api/v2/anchor/callback
+#   - name: vibrant
+#     type: noncustodial
+#     domain: vibrant.co
+#     callback_url: https://callback.vibrant.com/api/v2/anchor/callback
+#     signing_key: GA22WORKYRXB6AW7XR5GIOAOQUY4KKCENEAI34FN3KJNWHKDZTZSVLTU
+
+
+openapi: 3.0.0
+info:
+  title: Stellar Anchor Client Configuration API
+  version: "3.0"
+paths: {}
+components:
+  schemas:
+    ClientsConfig:
+      type: object
+      description: |-
+        Configuration for Stellar anchor clients, distinguishing between custodial and non-custodial clients.
+      properties:
+        custodial_clients:
+          type: array
```

Comment:
nit: List? Let's unify the terminology :)

Reply:
We usually use plural for list. Can you help pointing out the inconsistencies?

Reply:
My bad. I mean since it is described as `List` (List of custodial clients. Defaults to an empty list), shall we just use type `List` instead of `Array`?

Reply:
I do not see there is a type list in OpenAPI spec though. https://swagger.io/docs/specification/data-models/data-types/

Can you help to give me the reference?